thermodynamics/thermo_potential.py

```diff
@@ -0,0 +1,16 @@
+from pyiron_workflow import Workflow
+
+@Workflow.wrap.as_function_node()
```

Comment:
Super minor convenience: when you have nicely named return values (as you do here!) and don't need to specify output channel labels, the last couple versions of pyiron workflow will let you simply invoke `@Workflow.wrap.as_function_node` without the trailing `()` 😄